### PR TITLE
Update Sim Settlements 2

### PR DESCRIPTION
**Pull request by dm703**

### CODE REVIEW

**Comment by sibir-ine** [comment]
Unless there's a unifying theme (e.g., Bash Tags, cleaning data), unrelated edits to different mods (including related mods like patches or add-ons hosted on separate pages) should be made in separate PRs. In this case, the SS2 incompatibility changes would be made in one PR & the SS2 Chapter 2 addition in another.

**Comment by dm703** [comment]
Currently, my workflow is to delete my fork after each PR has been merged and start fresh.
This limits me to 1 PR per day (or so).
The unifying theme here is to get the masterlist updated with currently known info with respect to SS2.
SS2 is effectively an entire ecosystem (like Horizon or FROST).
Perhaps it should even have its own group (but that is probably a Discord discussion).
I agree that PRs that I need to submit that pertain to F04 areas other than SS2 need to be on separate PRs.

**Comment by sibir-ine** [comment]
If you create new branches off of your `v0.18` branch & submit pull requests from them, you can delete them after they're merged & just pull from upstream to update your `v0.18`. That way, you wouldn't need to delete your fork, & you could submit multiple PRs at the same time (merge conflicts can be resolved on the website fairly easily). If you need help with the process, we can walk you through it in the `#dev-questions` channel in the Discord.

**Comment by dm703** [comment]
Does that mean the effort in creating this PR has been wasted and this PR should be abandoned?

**Comment by sibir-ine** [comment]
No, you can just undo the changes for one of the mods in another commit & submit them in a separate PR.

File: masterlist.yaml [hunk 1]
```diff
@@ -3820,6 +3824,13 @@ plugins:
           - '[Sim Settlements 2 - Horizon Patch](https://www.nexusmods.com/fallout4/mods/54186/)'
         condition: 'active("Z_Horizon.esp") and not active("Z_Horizon_SS2_WastelandVenturers.esp")'
 
+  - name: 'SS2_XPAC_Chapter2.esm'
+    url:
+      - link: 'https://www.nexusmods.com/fallout4/mods/55817/'
+        name: 'Sim Settlements 2 - Chapter 2 (Nexus Mods)'
+      - link: 'https://bethesda.net/en/mods/fallout4/mod-detail/4247081/'
+        name: 'Sim Settlements 2 - Chapter 2 (Bethesda.net Mods)'
+
```
Comment (sibir-ine) [inline masterlist.yaml]: This can be removed from this PR since you've added it in #446.

```suggestion
```

**Review by sibir-ine — CHANGES_REQUESTED** [review]

**Review by sibir-ine — APPROVED** [review]

File: masterlist.yaml [hunk 2]
```diff
@@ -3752,7 +3752,11 @@ plugins:
         name: 'Sim Settlements 2 (Nexus Mods)'
       - link: 'https://bethesda.net/en/mods/fallout4/mod-detail/4183663/'
         name: 'Sim Settlements 2 (Bethesda.net Mods)'
-    inc: [ 'SimSettlements.esm' ]
+    inc:
+      - name: 'SimSettlements.esm'
+        display: 'Sim Settlements'
+      - name: 'Conquest.esp'
+        display: 'Conquest - Build New Settlements and Camping'
```
Comment (sibir-ine) [inline masterlist.yaml]: Sorry, forgot to check that the incompatibilities were alphabetically sorted:

```suggestion
      - name: 'Conquest.esp'
        display: 'Conquest - Build New Settlements and Camping'
      - name: 'SimSettlements.esm'
        display: 'Sim Settlements'
```

**Pull request merged**